docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+
+Gammapy started in 2013 and is now widely used in scientific publications. A proper citation scheme with correct authorship allows:
+
+- a proper citation of the used Gammapy release,
+- a proper recognition of the achieved work of any contributor,
+- compliance with the FAIR principles for Research Software (`FAIR4RS <https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs>`_).
+
+This PIG aims to set up the project policy about authorship for our citation scheme that is reminded here.
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Reminder of the citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should cite properly Gammapy.
+It is asked to cite the DOI of the latest Gammapy paper.
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This scheme and these two references are precise on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors, or `contributors`, that requires to be updated with time according to a
+general policy. This section defines what is a contributor to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (CI, GitHub configurations, gammapy-docs, release processes, etc),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, gammapy-cat, gammapy-recipes),
+- contributing to the maintenance of our repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
```

Comment:
This sentence is not clear to me. I guess it goes into the direction of "maintainer role" in the Gammapy project?

Reply:
This is a bit unclear. If this PIG defines two types of roles: contributors and maintainers, then this should be part of the second, right?

Reply:
The sentence is written such that the L.D. have some freedom.
The PIG of tasks, module responsibles, etc.. is not written. This is why I put this sentence per anticipation. 
If you have an other formulation, you're welcome;)

Reply:
This sentence is also too vague to be useful for me. If you want to go into the direction of long-term planning / steering the project, maybe this is clearer:

> contributing to the long-term project roadmap and over-all goals

?